docs/index.html

```diff
@@ -5488,54 +5488,32 @@ <h3 id="whats-the-deal-with-pagination-when-is-it-used-and-how-does-it-work">Wha
       <p>If your API uses cursor-based paging instead of an offset, you can use <code>z.cursor.get</code> and <code>z.cursor.set</code>:</p>
     </div>
     <div class="col-md-7 col-sm-12 col-height  docs-code">
-      <pre><code class="lang-js"><span class="hljs-comment">// the perform method of our trigger</span>
```

Comment:
Auto-generated HTML changes from the actual `.md` change. This will affect this website: https://zapier.github.io/zapier-platform/#paging